tasks/task_4/1_example_neutron_flux.py

```diff
@@ -89,6 +89,18 @@
 flux_slice.mean.shape = (mesh_width, mesh_height)
 
 fig = plt.subplot()
-plt.show(fig.imshow(flux_slice.mean))
 
+fig.imshow(flux_slice.mean).get_figure().savefig('flux_plot.png')
+try:
+    fig.imshow(flux_slice.mean).get_figure().savefig('/my_openmc_workshop/flux_plot.png')
```

Comment:
try combining the writing into the same try loop

Reply:
resolved